### ISSUE TITLE
Access error via sftp

### ISSUE THREAD

**Issue by fidegr** [AzuraCast/AzuraCast#5456]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### What happened?

We have detected another problem when accessing via sftp. When trying to access via filezilla or winscp it returns an access denied error. We have tried changing the password and it doesn't work either. Does anyone else have this problem?

### Relevant log output
```
azuracast  | {"level":"debug","time":"2022-06-03T11:10:01.142","sender":"dataprovider_bolt","message":"external auth hook executed, response code: 200"}
azuracast  | {"level":"debug","time":"2022-06-03T11:10:01.142","sender":"dataprovider_bolt","message":"external auth completed for user "toponautas", elapsed: 236.352883ms"}
azuracast  | {"level":"debug","time":"2022-06-03T11:10:01.142","sender":"connection_failed","client_ip":"77.224.237.170","username":"toponautas","login_type":"password","protocol":"SSH","error":"invalid external auth response: json: cannot unmarshal string into Go struct field User.quota_size of type int64"}
azuracast  | {"level":"debug","time":"2022-06-03T11:14:51.941","sender":"SSH","message":"connection refused, ip "77.224.237.170" is banned"}
```

**Comment by SC2Mitch** [comment]
Potential bug, can't reproduce locally on Filezilla however.

**Comment by Marcioghiraldelli** [comment]
did you manage to solve this problem?

**Comment by SlvrEagle23** [comment]
This was fixed in Rolling Release prior to 0.17.0 stable.
